src/credit-card/credit-card.ts

```diff
@@ -51,5 +51,6 @@ export function encrypt(cardNumber: string|number, cvv: string|number, month: st
   if(!validate(cardNumber, cvv, month, year)){
     return Observable.throw('Credit card details invalid');
   }
-  return creditCardEncrypt(cleanInput(cardNumber), cleanInput(cvv), Number(cleanInput(month)), Number(cleanInput(year)));
+  // allow CVV to be optional if it is null
+  return creditCardEncrypt(cleanInput(cardNumber), cvv === null ? '' : cleanInput(cvv), Number(cleanInput(month)), Number(cleanInput(year)));
```

Comment:
@OzzieOrca If `''` and `null` both work, could we use `null` to more semantically represent an intentionally missing value? Or are there other clients of this library that need it to be an empty string?